<?xml version="1.0" encoding="utf-8"?>
<launch>
    <arg name="veh" default="$(env VEHICLE_NAME)" />
    <arg name="config" default="default" />
    <arg name="param_file_name" default="default" />
    
    <group ns="$(arg veh)">
        <node name="predictive_perception_manager_node" 
              pkg="predictive_perception" 
              type="predictive_perception_manager_node.py" 
              output="screen" 
              clear_params="true">
            
            <rosparam file="$(find predictive_perception)/config/predictive_perception_manager_node/$(arg param_file_name).yaml" command="load"/>
            <param name="veh" value="$(arg veh)"/>
        </node>
    </group>
</launch>